languages/en.json

```diff
@@ -423,5 +425,14 @@
     "builder.mute.add.minutes.help": "Enter in minutes for how long you want to mute a member",
     "builder.mute.add.reason.title": "Reason of mute",
     "builder.mute.add.reason.help": "Enter a reason for muting member",
-    "builder.mute.add.ok": "Builder is complete. Member is muted"
+    "builder.mute.add.ok": "Builder is complete. Member is muted",
+
+    "builder.senddm.title.title": "Title of direct message",
+    "builder.senddm.title.help": "Enter title of direct message",
+    "builder.senddm.title": "Direct messages sender",
+    "builder.senddm.role.title": "Role for direct message",
+    "builder.senddm.role.help": "Enter name of the role, which you want to be selected for sending direct message",
+    "builder.senddm.message.title": "Content of direct message",
+    "builder.senddm.message.help": "Enter content, which will be sent in direct message",
+    "builder.senddm.ok": "Builder je complete. The message was sent"

```

Comment:
"je"? What does it mean?

Reply:
I thought i have seen that. Fixing it in second.